lib/ZumoOta/src/WebServerCustom.cpp

```diff
@@ -89,10 +130,10 @@ void WebServerCustom::init()
 
 void WebServerCustom::handleUploadRequest()
 {
-    server.on("/upload", HTTP_POST, [](AsyncWebServerRequest *request)
-    {
-        upload.handleUploadButtonPress();
-        request->send(200, "text/plain", "Upload Button gedrueckt");
+    server.on("/upload", HTTP_POST, [this](AsyncWebServerRequest *request) {
+    }, [this](AsyncWebServerRequest *request, String filename, size_t index, uint8_t *data, size_t len, bool final) {
+        // Verarbeite den Dateiupload

```

Comment:
another german easter egg ...

Reply:
It will be taken care of by the next commit.